test/e2e/specs/site-editor/preload.spec.js

```diff
@@ -46,6 +46,9 @@ test.describe( 'Preload', () => {
 		expect( requests ).toEqual( [
 			// Seems to be coming from `enableComplementaryArea`.
 			'/wp/v2/users/me',
+			// There are two separate settings OPTIONS requests. We should fix
+			// so the one for canUser and getEntityRecord are reused.
+			'/wp/v2/settings',
```

Comment:
I had to remove this in #69228 after merging #69173. I'm guessing the editor "is ready" a bit faster now, so the preloading test picks up `/wp/v2/settings` again.

This isn't a regression. As noted in the tests, both requests should be removed eventually.